assets/model_monitoring/components/model_monitor/model_monitor_compute_histogram/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: spark
 name: model_monitor_compute_histogram
 display_name: Model Monitor - Compute Histogram
 description: Compute a histogram given an input data and associated histogram buckets.
-version: 0.3.4
+version: 0.3.5
```

Comment:
Did you make changes in the compute_histogram component? I thought you only changed the compute_histogram_bucket? Let me know if it is not the case.

Reply:
True, my changes affected histogram utils but not necessarily the ones used by compute histograms so I think we can revert.